src/cgame/cg_rocket_datasource.cpp

```diff
@@ -936,7 +936,27 @@ void CG_Rocket_BuildPlayerList( const char* )
 
 		Info_SetValueForKey( buf, "num", va( "%d", score->client ), false );
 		Info_SetValueForKey( buf, "score", va( "%d", score->score ), false );
-		Info_SetValueForKey( buf, "ping", va( "%d", score->ping ), false );
+
+		const char* B = Info_ValueForKey( CG_ConfigString( CS_SERVERINFO ),"B" );
+		const char bot_status = B[ score->client ];
+
+		// HACK: Display bot status in “Ping” column.
+		if ( bot_status == '-' )
+		{
+			// This player is not a bot.
+			Info_SetValueForKey( buf, "ping", va( "%d", score->ping ), false );
+		}
+		else if ( bot_status >= '0' && bot_status <= '9' )
+		{
+			// Bot skill.
+			Info_SetValueForKey( buf, "ping", va( "S%c", bot_status ), false );
```

Comment:
Maybe we could set this to "sk%c", I think it would be more obvious

```suggestion
			Info_SetValueForKey( buf, "ping", va( "sk%c", bot_status ), false );
```

Reply:
Done.